packages/devtools_app/lib/src/shared/console/eval/inspector_tree_v2.dart

```diff
@@ -54,14 +58,12 @@ class InspectorTreeNode {
     if (dirty) {
       _isDirty = true;
       _shouldShow = null;
-      if (_childrenCount == null) {
-        // Already dirty.
-        return;
-      }
-      _childrenCount = null;
       if (parent != null) {
         parent!.isDirty = true;
       }
+      if (whenDirty != null) {
+        whenDirty!(this);
+      }
```

Comment:
this can be condensed `whenDirty?.call(this)`

Reply:
Done!